File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while(height <= 0 || height > 8);

    for(int i = 1; i <= height; i++)
    {
        for(int j = height - i; j <= 0; j++)
        {
            printf("%0*d", j, ' ');
        }
        printf("%0*d\n", i, '#');
    }
}

File: HarvardX/sentimental-mario-less/mario.py
from cs50 import get_int

# get height from user input between 1 and 8 inclusive
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    # print spaces
    for j in range(height-i-1):
        print(" ", end="")
    # print dashes
    for j in range(i+1):
        print("#", end="")
    print()